app/controlplane/api/controlplane/v1/workflow.proto

```diff
@@ -72,7 +72,8 @@ message WorkflowServiceListResponse {
 }
 
 message WorkflowServiceViewRequest {
-  string id = 1 [(buf.validate.field).string.uuid = true];
+  optional string id = 1;
```

Comment:
You might want to add this validation. BTW why did you decide to make them optional? To me optional is useful when you want to differentiate between having an empty value or not-set value. In this case, it doesn't make a difference right?

https://github.com/chainloop-dev/chainloop/blob/4dac4f6150ccb9d42574c7d9e878401a414784ac/app/controlplane/api/controlplane/v1/workflow_contract.proto#L59

Reply:
Checking that example from workflow contracts.